### DISCUSSION TITLE
Add additional reset system permissions to: created by current user

### DISCUSSION THREAD

**Discussion by martinomg** [directus/directus#13766] [Feature Requests]
I think it would be a huge time saver to have a fast way to set system permission defaults for CRUD and share only for items created by current user. This applies for almost any app where the user needs to handle it's own data.

Basically this would mean replicating for all the collections the following system permission selection sequence:
1. Select an action (eg. READ)
2. Permission > Use Custom
3. Item permissions > Add filter > Set filter to: User Created **Equals** $CURRENT_USER

Replicating this for all CRUD operations and a big amount of collections is a lot of time.

This could be an additional option to the already existing "App access minimum" and "Recommended defaults"

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
